.github/hooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+cd "$(git rev-parse --show-toplevel)"
+ESLINT="node_modules/.bin/eslint"
+
+STAGED_FILES=($(git diff --cached --name-only --diff-filter=ACM | grep ".jsx\{0,1\}$"))
+
+if [[ "$STAGED_FILES" = "" ]]; then
+  exit 0
+fi
+
+failed=0
+for file in ${STAGED_FILES}; do
+  git show :$file | eslint $file
+  if [[ $? != 0 ]] ; then
+    failed=1
```

Comment:
break the loop for efficiency

Reply:
Why? Let it check all files and show all errors, instead of only first one.

Reply:
Because this script does not show errors, only "🚫🚫🚫 ESLint failed, git commit denied!"

Reply:
As far as I understand it, it doesn't block Eslint output, meaning before my custom error messages there should be lint output for each error

Reply:
Then, that is ok